thesis/conclusion.tex

```diff
@@ -0,0 +1,31 @@
+\chapter{Conclusion}
+This bachelor project was commenced four month ago.
+The goal was to develop an application platform for badminton players, coaches and consultants that enables them to monitor and track their performances.
+This was done by developing a cross-platform mobile application for \textit{Android} and \textit{iOS} and a server application to handle data storage.
+
+When defining requirements for the product, we distributed a user survey with social media and collected responses from $529$ individuals.
+This gave us an incredible insight into the user segments.
+Combined with the consultant interview we have clear expectations of what to do and what \textit{not} to do.
+This was a great experience to validate our expected requirements with those of the users.
+
+The development work flow has been a great success.
+We have exercised modern development methodologies and have great benefits of working agile and systematically.
+From task grooming, sprint planning and retrospection made the work progress clear and understandable.
+Task reviews before merging \textit{Git} branches helped keeping the code quality high as well as it gave the individual developers understanding of the overall structure and movements of the code.
+
+In the process of development, it was decided to focus on new technologies, aware of the added risks, for the technology stacks.
+This caused us to have several problems with the front-end application framework \glsi{nativescript} with bad documentation, non-compliant build and test configurations and more.
+This was a real setback on the technology choice when considering the importance of tests and documentation.
+
+On the back-end, the technologies were more stable and proved to be a great success.
+The \gls{api} specification \glsi{graphql} is a powerful concept with fast development and auto-generated interactive documentation.
+It enabled us to develop the back-end with a clear structure and let it be ignorant of the client uses.
+
+The final applications did not meet all of our requirements and we have identified several factors why this is the case.
+The development of the racket sensor was not completed in time, why all interaction with the racket and views regarding these actions have not been completed.
+Further more, the expected views, according to the roadmap, were not what was needed, so others, and more, were made and the roadmap was not accurate any more.
```

Comment:
Proposal:   
Further more, the expected amount of views on the front-end application did not prove accurate. Several other views and navigation management was needed that we did not account for in the beginning when the roadmap was created.